index.js

```diff
@@ -146,8 +165,43 @@ module.exports = function(app) {
         }
       ]
     }
-    app.signalk.addDelta(delta)
+    debug('Sending path zones notification:', JSON.stringify(notificationDelta))
+    app.signalk.addDelta(notificationDelta)
```

Comment:
It was like this by me, but should use [handleMessage](https://demo.signalk.org/documentation/develop/plugins/deltas.html?highlight=handleMessage#sending-deltas) that is the documented way to send deltas.

Reply:
Ok. I'll look at it.